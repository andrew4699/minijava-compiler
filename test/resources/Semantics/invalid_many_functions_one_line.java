class foo {

  public static void main(String[] args) {
    System.out.println(new bar().buzz());
  }
}

class bar extends foo {
  public int buzz() {
    return this.one()
        + this.two(2)
        + this.three(3)
        + this.oops();
  }
  public int one() {
    return 1;
  }
  public int two(int t) {
    return t;
  }
  public int three(int t) {
    return t;
  }
}