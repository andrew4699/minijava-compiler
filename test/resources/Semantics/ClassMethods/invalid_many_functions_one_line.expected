[ClassMethodsVisitor - 13]: oops() does not belong to class bar
